main.go

```diff
@@ -277,21 +277,24 @@ func main() {
 		return
 	}
 
-	originPath := flag.Arg(0)
+	originPaths := flag.Args()
 
 	if filePath != "" {
-		deprecatedMessagesCh <- fmt.Sprintf("-%s is deprecated. Put file name as last argument to the command(Example: goimports-reviser -rm-unused -set-alias -format goimports-reviser/main.go)", filePathArg)
-		originPath = filePath
+		deprecatedMessagesCh <- fmt.Sprintf("-%s is deprecated. Put file name(s) as last argument to the command(Example: goimports-reviser -rm-unused -set-alias -format goimports-reviser/main.go)", filePathArg)
+		originPaths = append(originPaths, filePath)
 	}
 
-	if originPath == "" {
-		originPath = reviser.StandardInput
+	var originPath string
+	if len(originPaths) == 0 || (len(originPaths) == 1 && originPaths[0] == "-") {
+		originPaths[0] = reviser.StandardInput
 	}
 
-	if err := validateRequiredParam(originPath); err != nil {
-		fmt.Printf("%s\n\n", err)
-		printUsage()
-		os.Exit(1)
+	for _, originPath = range originPaths {
+		if err := validateRequiredParam(originPath); err != nil {
+			fmt.Printf("%s\n\n", err)
```

Comment:
You are using log.Fatal to handle other errors

log package use stdErr when nothing is configured

https://cs.opensource.google/go/go/+/refs/tags/go1.23.3:src/log/log.go;l=87


Here your code use, stdOut

I would suggest you using this

```suggestion
			log.Printf("%s\n\n", err)
```